interface/patient_file/report/custom_report.php

```diff
@@ -844,7 +844,7 @@ function next_prev(action){
                   else {
                     echo "<img src='" . $GLOBALS['webroot'] .
                       "/controller.php?document&retrieve&patient_id=&document_id=" .
-                      $document_id . "&as_file=false'><br><br>";
+                      $document_id . "&as_file=false&original_file=true&disable_exit=false&show_original=true'><br><br>";
```

Comment:
I tried it but if I don’t send all the variables in the url ‘Document class’ put true in the ‘original_file’ (first variable) instead ‘show_original’, current logic doesn’t check a variable name..